docs/aranya-mvp.md

```diff
@@ -247,7 +247,13 @@ device for use with AFC. If the address already exists for this device, it is re
 address. Capable of resolving addresses via DNS. For use with CreateChannel and receiving messages.
 Can take either DNS name, IPv4, or IPv6. Current implementation uses a bidi map, so we can reverse
 lookup.
-- `UnsetAfcNetIdentifier(team_id, device_id, net_identifier)` - disassociate a network address from a device.
+- `UnsetAfcNetIdentifier(team_id, device_id, net_identifier)` - disassociate a AFC network address from a device.
+- `SetAqcNetIdentifier(team_id, device_id, net_identifier)` - associate a network address to a
```

Comment:
For AQC and AFC we should _probably_ ensure that the device ID <-> net ID is 1:1.

However, this does _not_ preclude multiple DNS names from resolving to the same IP address (obviously, since that is the entire purpose of using DNS names).

Reply:
What I meant is that if device A has the net identifier "foo.com" then device B cannot _also_ have the net identifier "foo.com". IOW, net identifiers are globally unique within a team.

I kinda can see valid use cases for duplicate net identifiers, but until we have a concrete use case we should not support it in the MVP.